pkgs/top-level/aliases.nix

```diff
@@ -737,7 +737,6 @@ mapAliases ({
   mpd_clientlib = libmpdclient; # Added 2021-02-11
   mpich2 = mpich; # Added 2018-08-06
   msf = metasploit; # Added 2018-04-25
-  multimc = throw "multimc was removed from nixpkgs; use polymc instead (see https://github.com/NixOS/nixpkgs/pull/154051 for more information)"; # Added 2022-01-08
```

Comment:
Should be restored. Could add a note to view the release notes instead of linking to the PR.